Triton_Inference_Server_Python_API/examples/kafka-io/start-kafka.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+export DEBIAN_FRONTEND=noninteractive
+
+wget https://dlcdn.apache.org/kafka/3.7.0/kafka_2.13-3.7.0.tgz
+tar -xzf kafka_2.13-3.7.0.tgz
+cd kafka_2.13-3.7.0
+
+echo "Setting up JAVA 17"
+apt-get update -q -y
+apt-get upgrade -q -y
```

Comment:
```suggestion
```
Let's remove this as it takes a very long time. We are only updating in the first place because we want to install java which will be available for install after `apt-get update`

Reply:
Removed the line